lib/multer.js

```diff
@@ -0,0 +1,14 @@
+const multer = require('multer');
+const path = require('path');
+
+const storage = multer.diskStorage({
+  destination(_req, _file, cb) {
+    cb(null, path.join(__dirname, '../public/uploads/'));
```

Comment:
How are we currently managing access to this directory? Is it only accessible by our api?

Reply:
Nope, currently it's publically accessible, but since we are going to use the distributed file system, so code gonna change accordingly.